include/wx/gtk/webview_webkit.h

```diff
@@ -216,9 +212,18 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewFactoryWebKit : public wxWebViewFactory
                               const wxSize& size = wxDefaultSize,
                               long style = 0,
                               const wxString& name = wxASCII_STR(wxWebViewNameStr)) override
-    { return new wxWebViewWebKit(parent, id, url, pos, size, style, name); }
+    { 
+        wxWebView* webView = new wxWebViewWebKit;
+        if (webView->Create(parent, id, url, pos, size, style, name))
+            return webView;
+        else
+            return nullptr;
```

Comment:
Shouldn't we `delete webView` here (or, rather, use `std::unique_ptr<>` for it in the first place)?

Reply:
Changed to use `std::unique_ptr`